mypyc/codegen/emitwrapper.py

```diff
@@ -741,8 +708,8 @@ class WrapperGenerator:
 
     # TODO: Use this for more wrappers
 
-    def __init__(self, cl: ClassIR, emitter: Emitter) -> None:
-        self.cl = cl
+    def __init__(self, cl_name_prefix: str, emitter: Emitter) -> None:
```

Comment:
Hmm, I am thinking that `cl: Optional[ClassIR]` and then do `self.cl.name_prefix(self.emitter.names) if self.cl else None` is better